src/sentry/dynamic_sampling/tasks/recalibrate_orgs.py

```diff
@@ -36,17 +26,6 @@
 from sentry.silo import SiloMode
 from sentry.tasks.base import instrumented_task
 
-# Since we are using a granularity of 60 (minute granularity), we want to have a higher time upper limit for executing
-# multiple queries on Snuba.
-RECALIBRATE_ORGS_MAX_SECONDS = 600
-
-
-class RecalibrationError(Exception):
```

Comment:
Removed the verbose error handling in favor of simply propagating the errors to the logger. Might need in the future to unify how we want to report errors across the ds tasks.